app.js

```diff
@@ -22,12 +22,13 @@ i18next
       loadPath: __dirname + '/locales/{{lng}}/{{ns}}.json',
       addPath: __dirname + '/locales/{{lng}}/{{ns}}.missing.json'
     },
+    preload: ['en', 'es'],
```

Comment:
Are you suggesting this for some reason? The issue is solved just with the `cookieDomain` being correctly set.

Reply:
![image](https://user-images.githubusercontent.com/1086194/70392016-851ca280-19db-11ea-823b-ed7839eeb09c.png)
https://www.i18next.com/overview/configuration-options